config.yml

```diff
@@ -0,0 +1,71 @@
+version: 2.1
```

Comment:
Did you mean to check this file in since the CircleCI config already exists in `.circleci/config.yml`?

Reply:
Just an oversight, I'd started this before a meeting and then came back to it.